src/doc/tutorials/PythonExpressions.rst

```diff
@@ -0,0 +1,202 @@
+.. _Tutorials_Python_Expressions:
+
+Python Expressions
+==================
+
+.. toctree::
+       :maxdepth: 2
+
+This tutorial describes using Python Expressions.
+
+Python expressions overview
+---------------------------
+
```

Comment:
I think its worth adding some motivation about *why* and/or *when* someone might want to use Python expressions. And, I think we could benefit from at least one link from here back to the [section on expressions](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/using_visit/Quantitative/Expressions.html#expressions)

Maybe...

When the available expressions fail to provide needed functionality, users can extend VisIt's expression system in arbitrary ways using Python Expressions. However, Python Expressions require users to become somewhat converscent in detailed aspects of how mesh and variable data is represented in terms of VTK objects as well as how they are decomposed and processed in parallel. Nonetheless, Python expressions provide a powerful approach to filling in gaps in needed functionality.